src/test/librados/TestCase.cc

```diff
@@ -343,8 +344,8 @@ void RadosTestPP::cleanup_namespace(librados::IoCtx ioctx, std::string ns)
     ioctx.locator_set_key(it->get_locator());
     ObjectWriteOperation op;
     op.remove();
-    librados::AioCompletion *completion = s_cluster.aio_create_completion();
-    ASSERT_EQ(0, ioctx.aio_operate(it->get_oid(), completion, &op,
+    std::unique_ptr<librados::AioCompletion> completion(s_cluster.aio_create_completion());
```

Comment:
a single `release()` is good enough, and we already have it. what we are missing is the error handling of `ASSERT_EQ()`, i'd suggest use a `unique_ptr<>` or `intrusive_ptr<>` here if we want to have `ASSERT_EQ()` in `TearDown()`.